include/utils/ignore_warnings.h

```diff
@@ -32,8 +32,12 @@
 #pragma clang diagnostic ignored "-Wnested-anon-types"
 #pragma clang diagnostic ignored "-Wsign-compare"
 #pragma clang diagnostic ignored "-Wunused-private-field"
+#pragma clang diagnostic ignored "-Wsign-compare"
+#pragma clang diagnostic ignored "-Wextra"
+#if (__clang_major__ > 3) || ((__clang_major__) == 3 && (__clang_minor__ >= 5))
```

Comment:
Sorry, should have caught this before, the parens on the second clause seem off, maybe:
`#if (__clang_major__ > 3) || ((__clang_major__ == 3) && (__clang_minor__ >= 5))`

Reply:
I agree the paren is off, but I don't think it matters because `==` takes precedence over `&&` anyway.

Reply:
Yeah, those parens just don't do anything.
Also, it looks like `-Winconsistent-missing-override` doesn't work on clang 3.5.1 which is our current minimum clang version.
I couldn't pinpoint what clang version it was added. I saw the commit [here](https://reviews.llvm.org/rL218925) but not sure when it made it in

Reply:
I've seen a [couple](https://github.com/CGAL/cgal/issues/2026)  [sources](https://stackoverflow.com/questions/29145476/requiring-virtual-function-overrides-to-use-override-keyword) that say "since 3.5" but since that's apparently not quite right let's try the next point release and restrict that line to clang 3.6.0 and greater?

Reply:
Looks like it went in on 3.6.0
[3.5.2](https://llvm.org/svn/llvm-project/cfe/tags/RELEASE_352/final/include/clang/Basic/DiagnosticGroups.td) does not have it
but [3.6.0](https://llvm.org/svn/llvm-project/cfe/tags/RELEASE_360/final/include/clang/Basic/DiagnosticGroups.td) does